user-docs/pupil-capture.md

```diff
@@ -209,67 +224,190 @@ If you open up a session folder you will see a collection of video(s) and data f
 
+<aside class="notice">
 For this to work your network needs to allow `UDP` transport. If the nodes do not find each other, create a local wifi network and use that instead.
+</aside>
 
 <aside class="notice">
   Pupil Groups can easily be integrated in your own app or device. Have a look at <a href="https://github.com/pupil-labs/pupil-helpers/tree/master/pupil_sync">pupil helpers</a> to get started.
 </aside>
 
-### Streaming Pupil Data over the network
+<!-- ### Streaming Pupil Data over the network
 
-> {{< video-webm src="/videos/pupil-remote/pr.webm" >}}
+Its network interface is based on the ZeroMQ [`REQ-REP` pattern](http://zguide.zeromq.org/php:chapter1#Ask-and-Ye-Shall-Receive).
+The plugin opens the "Reply" socket and waits for client connections. The Pupil Remote protocol consists mostly of single messages.
 
-`Pupil Remote` is a plugin that is used to broadcast data over the network using the excellent library [Zero MQ](http://zeromq.org/).
+> ```
+Send simple string messages to control Pupil Capture functions:
+    'R' start recording with auto generated session name
+    'R rec_name' start recording and name new session name: rec_name
+    'r' stop recording
+    'C' start currently selected calibration
+    'c' stop currently selected calibration
+    'T 1234.56' Timesync: make timestamps count form 1234.56 from now on.
+    't' get pupil capture timestamp returns a float as string.
 
-* Load the `Pupil Remote` plugin from the `General` sub-menu in the GUI (it is loaded by default).
-* It will automatically begin broadcasting at the default `Address` specified.
-* Change the address and port as desired.
-* If you want to change the address, just type in the address after the `tcp://`
 
-### Receiving Data with your own app
-ZeroMQ has bindings to many languages. Reading the stream using python goes like so:
+    # IPC Backbone communication
+    'PUB_PORT' return the current pub port of the IPC Backbone
+    'SUB_PORT' return the current sub port of the IPC Backbone
+
+Mulitpart messages conforming to pattern:
+    part1: 'notify.' part2: a msgpack serialized dict with at least key 'subject':'my_notification_subject'
+    will be forwared to the Pupil IPC Backbone.
+```
+
+The "IPC Backbone" is the name for the built-in broadcast functionality. You will need to request the `SUB_PORT`
+most frequently to be able to subscribe to the published data.
+
+See the pupil-helper [`filter_messages.py`](https://github.com/pupil-labs/pupil-helpers/blob/master/pupil_remote/filter_messages.py)
+script for an example on how to subscribe to a specific topic.
+
+
+### "IPC Backbone" Message Format
+
+All messages that are broadcasted by Pupil Capture comply with the following format:
+* Messages are at least two-part containing the topic and the payload
+* The payload is a [msgpack](http://msgpack.org/index.html) serialized dictionary
```

Comment:
Topic is a string.

Consider: simple example of topic and payload before diving into the script.